task.py.6.py

```diff
@@ -0,0 +1,34 @@
+Phone_Book = {
+    "Bob": "677545",
+    "Marg": "345888",
+    "Clif": "989000",
+}
+while True:
+
+    command = input('enter the command:')
+
+    if command == 'stats':
+        print(f'{len(Phone_Book)} - number of subscribers')
+    if command == 'add':
+        print('enter the name:')
+        name = input("> ")
+        print('enter the phone number')
+        number = input("> ")
+        Phone_Book[name] = number
```

Comment:
It is not allowed to override records in the phonebook. Please, add here a check for existing key and add a record only if key does not exist